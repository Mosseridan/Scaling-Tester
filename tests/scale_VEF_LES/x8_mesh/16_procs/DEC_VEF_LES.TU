Statistiques de post resolution

Temps total                       9.07442

