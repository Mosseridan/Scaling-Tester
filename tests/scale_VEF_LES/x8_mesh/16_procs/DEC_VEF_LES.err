Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VEF_LES
feenableexcept enabled.
Data file : DEC_VEF_LES
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VEF_LES/x8_mesh/16_procs
Nom du cas DEC_VEF_LES
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
It treats a calculation with a precision 1e-05
It deals with a calculation using precision format 5
Execution of the interpreter Lire_Tgrid
Reading a mesh which comes from Tgrid
Association of the domain on which the interpreter applies Lire_Tgrid
Reading of the file Tube_coude_463259.msh ...
The total number of elements to read is 463259
3D elements of type Tetrahedron
Reading a comment:
(0

Reading of the dimension of the case:
Dimension 3.

Reading a comment:
(0

The total number of nodes to read is 89710

89710 nodes are read in the area 8


The type of area 9 is 1 (1:fluid, 17:solid)

The total number of faces to read is 950399

Reading a comment:
(0

902637 internal faces are read in the area 10

Reading a comment:
(0

19438 faces are read from the boundary number 11

Reading a comment:
(0

27656 faces are read from the boundary number 12

Reading a comment:
(0

334 faces are read from the boundary number 13

Reading a comment:
(0

334 faces are read from the boundary number 14

Reading a comment:
(0

Reading of a name:
The area 9 is called fluid

Reading of a name:
The area 10 is called interieur

Reading of a name:
The area 11 is called symmetry

Reading of a name:
The area 12 is called wall

Reading of a name:
The area 13 is called outlet

Reading of a name:
The area 14 is called inlet

Elimination of 0 unnecessary nodes in the domain dom
Execution of the interpreter Dilate
Association of the domain on which the interpreter applies Dilate
Expands the domain dom of a factor 0.001
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x
Interpretation of the function x Ok
Reading and interpretation of the function y
Interpretation of the function y Ok
Reading and interpretation of the function -z
Interpretation of the function -z Ok
Execution of the interpreter ExtrudeBord
Association of the domain on which the interpreter applies ExtrudeBord
number of 3D nodes :89710
number of 3D faces of inlet :334
number of nodes per faces in 3D :3
boundary maintained symmetry
boundary maintained wall
boundary to eliminate outlet
Execution of the interpreter Raffiner_Simplexes
Association of the domain on which the interpreter applies Raffiner_Simplexes
Execution of the interpreter Raffiner_Simplexes
Association of the domain on which the interpreter applies Raffiner_Simplexes
Decouper : Splitting of a domain
 Domain name to split : box
==============================================
The extreme coordinates of the domain box are:
x is between -0.5334 and -0.4572
y is between -0.1624 and -0.1524
z is between 0 and 0.0381
==============================================
==============================================
The volume cells of the domain box are:
sum(volume cells)= 2.27368e-05
mean(volume cells)= 1.51951e-10
min(volume cells)= 6.8383e-11
max(volume cells)= 6.01739e-10
==============================================
domaine = box
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 5799
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 2
The partitioner has generated 16 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 16 parts:
 Construction of part number 0
  Number of nodes    : 1939
  Number of elements : 9352
  Number of joints   : 5
  Joint 0 PeVoisin 1 NbSommets 104 NbFaces 176 NbElemDist 2173
  Joint 1 PeVoisin 2 NbSommets 48 NbFaces 70 NbElemDist 859
  Joint 2 PeVoisin 3 NbSommets 130 NbFaces 216 NbElemDist 3491
  Joint 3 PeVoisin 6 NbSommets 41 NbFaces 56 NbElemDist 972
  Joint 4 PeVoisin 7 NbSommets 57 NbFaces 88 NbElemDist 1230
               Total:    NbSommets 380 NbFaces 606 NbElemDist 8725
Writing part 0 into the binary file box_0000.Zones
Writing of 1939 nodes.
Writing of 9352 elements.
 Construction of part number 1
  Number of nodes    : 1949
  Number of elements : 9352
  Number of joints   : 3
  Joint 0 PeVoisin 0 NbSommets 104 NbFaces 176 NbElemDist 2432
  Joint 1 PeVoisin 2 NbSommets 160 NbFaces 284 NbElemDist 3046
  Joint 2 PeVoisin 3 NbSommets 67 NbFaces 105 NbElemDist 1515
               Total:    NbSommets 331 NbFaces 565 NbElemDist 6993
Writing part 1 into the binary file box_0001.Zones
Writing of 1949 nodes.
Writing of 9352 elements.
 Construction of part number 2
  Number of nodes    : 1975
  Number of elements : 9352
  Number of joints   : 6
  Joint 0 PeVoisin 0 NbSommets 48 NbFaces 70 NbElemDist 1081
  Joint 1 PeVoisin 1 NbSommets 160 NbFaces 284 NbElemDist 3153
  Joint 2 PeVoisin 3 NbSommets 39 NbFaces 57 NbElemDist 1101
  Joint 3 PeVoisin 5 NbSommets 70 NbFaces 110 NbElemDist 1692
  Joint 4 PeVoisin 7 NbSommets 69 NbFaces 108 NbElemDist 1599
  Joint 5 PeVoisin 14 NbSommets 116 NbFaces 200 NbElemDist 2531
               Total:    NbSommets 502 NbFaces 829 NbElemDist 11157
Writing part 2 into the binary file box_0002.Zones
Writing of 1975 nodes.
Writing of 9352 elements.
 Construction of part number 3
  Number of nodes    : 1925
  Number of elements : 9352
  Number of joints   : 5
  Joint 0 PeVoisin 0 NbSommets 130 NbFaces 216 NbElemDist 3911
  Joint 1 PeVoisin 1 NbSommets 67 NbFaces 105 NbElemDist 1618
  Joint 2 PeVoisin 2 NbSommets 39 NbFaces 57 NbElemDist 811
  Joint 3 PeVoisin 6 NbSommets 78 NbFaces 128 NbElemDist 1518
  Joint 4 PeVoisin 7 NbSommets 36 NbFaces 44 NbElemDist 1019
               Total:    NbSommets 350 NbFaces 550 NbElemDist 8877
Writing part 3 into the binary file box_0003.Zones
Writing of 1925 nodes.
Writing of 9352 elements.
 Construction of part number 4
  Number of nodes    : 1980
  Number of elements : 9352
  Number of joints   : 5
  Joint 0 PeVoisin 5 NbSommets 155 NbFaces 269 NbElemDist 2964
  Joint 1 PeVoisin 6 NbSommets 95 NbFaces 158 NbElemDist 2118
  Joint 2 PeVoisin 7 NbSommets 102 NbFaces 170 NbElemDist 1936
  Joint 3 PeVoisin 8 NbSommets 3 NbFaces 0 NbElemDist 196
  Joint 4 PeVoisin 9 NbSommets 106 NbFaces 178 NbElemDist 1988
               Total:    NbSommets 461 NbFaces 775 NbElemDist 9202
Writing part 4 into the binary file box_0004.Zones
Writing of 1980 nodes.
Writing of 9352 elements.
 Construction of part number 5
  Number of nodes    : 1974
  Number of elements : 9352
  Number of joints   : 6
  Joint 0 PeVoisin 2 NbSommets 70 NbFaces 110 NbElemDist 1323
  Joint 1 PeVoisin 4 NbSommets 155 NbFaces 269 NbElemDist 3214
  Joint 2 PeVoisin 7 NbSommets 87 NbFaces 141 NbElemDist 2336
  Joint 3 PeVoisin 8 NbSommets 151 NbFaces 267 NbElemDist 3180
  Joint 4 PeVoisin 9 NbSommets 29 NbFaces 32 NbElemDist 656
  Joint 5 PeVoisin 14 NbSommets 80 NbFaces 134 NbElemDist 1448
               Total:    NbSommets 572 NbFaces 953 NbElemDist 12157
Writing part 5 into the binary file box_0005.Zones
Writing of 1974 nodes.
Writing of 9352 elements.
 Construction of part number 6
  Number of nodes    : 1964
  Number of elements : 9352
  Number of joints   : 4
  Joint 0 PeVoisin 0 NbSommets 41 NbFaces 56 NbElemDist 947
  Joint 1 PeVoisin 3 NbSommets 78 NbFaces 128 NbElemDist 1532
  Joint 2 PeVoisin 4 NbSommets 95 NbFaces 158 NbElemDist 1642
  Joint 3 PeVoisin 7 NbSommets 182 NbFaces 316 NbElemDist 3589
               Total:    NbSommets 396 NbFaces 658 NbElemDist 7710
Writing part 6 into the binary file box_0006.Zones
Writing of 1964 nodes.
Writing of 9352 elements.
 Construction of part number 7
  Number of nodes    : 1954
  Number of elements : 9352
  Number of joints   : 6
  Joint 0 PeVoisin 0 NbSommets 57 NbFaces 88 NbElemDist 1319
  Joint 1 PeVoisin 2 NbSommets 69 NbFaces 108 NbElemDist 1558
  Joint 2 PeVoisin 3 NbSommets 36 NbFaces 44 NbElemDist 1283
  Joint 3 PeVoisin 4 NbSommets 102 NbFaces 170 NbElemDist 2030
  Joint 4 PeVoisin 5 NbSommets 87 NbFaces 141 NbElemDist 1886
  Joint 5 PeVoisin 6 NbSommets 182 NbFaces 316 NbElemDist 4047
               Total:    NbSommets 533 NbFaces 867 NbElemDist 12123
Writing part 7 into the binary file box_0007.Zones
Writing of 1954 nodes.
Writing of 9352 elements.
 Construction of part number 8
  Number of nodes    : 1962
  Number of elements : 9352
  Number of joints   : 7
  Joint 0 PeVoisin 4 NbSommets 3 NbFaces 0 NbElemDist 199
  Joint 1 PeVoisin 5 NbSommets 151 NbFaces 267 NbElemDist 3162
  Joint 2 PeVoisin 9 NbSommets 128 NbFaces 221 NbElemDist 2631
  Joint 3 PeVoisin 10 NbSommets 65 NbFaces 104 NbElemDist 1462
  Joint 4 PeVoisin 11 NbSommets 85 NbFaces 137 NbElemDist 1901
  Joint 5 PeVoisin 14 NbSommets 57 NbFaces 87 NbElemDist 1388
  Joint 6 PeVoisin 15 NbSommets 47 NbFaces 64 NbElemDist 1222
               Total:    NbSommets 536 NbFaces 880 NbElemDist 11965
Writing part 8 into the binary file box_0008.Zones
Writing of 1962 nodes.
Writing of 9352 elements.
 Construction of part number 9
  Number of nodes    : 1940
  Number of elements : 9352
  Number of joints   : 5
  Joint 0 PeVoisin 4 NbSommets 106 NbFaces 178 NbElemDist 2187
  Joint 1 PeVoisin 5 NbSommets 29 NbFaces 32 NbElemDist 752
  Joint 2 PeVoisin 8 NbSommets 128 NbFaces 221 NbElemDist 2932
  Joint 3 PeVoisin 10 NbSommets 13 NbFaces 16 NbElemDist 308
  Joint 4 PeVoisin 11 NbSommets 97 NbFaces 159 NbElemDist 1921
               Total:    NbSommets 373 NbFaces 606 NbElemDist 8100
Writing part 9 into the binary file box_0009.Zones
Writing of 1940 nodes.
Writing of 9352 elements.
 Construction of part number 10
 Adding of a new joint : 14
  Number of nodes    : 1951
  Number of elements : 9352
  Number of joints   : 7
  Joint 0 PeVoisin 8 NbSommets 65 NbFaces 104 NbElemDist 1295
  Joint 1 PeVoisin 9 NbSommets 13 NbFaces 16 NbElemDist 240
  Joint 2 PeVoisin 11 NbSommets 203 NbFaces 354 NbElemDist 4428
  Joint 3 PeVoisin 12 NbSommets 48 NbFaces 66 NbElemDist 1369
  Joint 4 PeVoisin 13 NbSommets 82 NbFaces 133 NbElemDist 1829
  Joint 5 PeVoisin 14 NbSommets 0 NbFaces 0 NbElemDist 27
  Joint 6 PeVoisin 15 NbSommets 119 NbFaces 198 NbElemDist 2650
               Total:    NbSommets 530 NbFaces 871 NbElemDist 11838
Writing part 10 into the binary file box_0010.Zones
Writing of 1951 nodes.
Writing of 9352 elements.
 Construction of part number 11
  Number of nodes    : 2005
  Number of elements : 9352
  Number of joints   : 7
  Joint 0 PeVoisin 8 NbSommets 85 NbFaces 137 NbElemDist 1577
  Joint 1 PeVoisin 9 NbSommets 97 NbFaces 159 NbElemDist 2171
  Joint 2 PeVoisin 10 NbSommets 203 NbFaces 354 NbElemDist 4542
  Joint 3 PeVoisin 12 NbSommets 35 NbFaces 49 NbElemDist 834
  Joint 4 PeVoisin 13 NbSommets 22 NbFaces 29 NbElemDist 399
  Joint 5 PeVoisin 14 NbSommets 1 NbFaces 0 NbElemDist 42
  Joint 6 PeVoisin 15 NbSommets 36 NbFaces 53 NbElemDist 546
               Total:    NbSommets 479 NbFaces 781 NbElemDist 10111
Writing part 11 into the binary file box_0011.Zones
Writing of 2005 nodes.
Writing of 9352 elements.
 Construction of part number 12
  Number of nodes    : 1959
  Number of elements : 9352
  Number of joints   : 4
  Joint 0 PeVoisin 10 NbSommets 48 NbFaces 66 NbElemDist 1109
  Joint 1 PeVoisin 11 NbSommets 35 NbFaces 49 NbElemDist 919
  Joint 2 PeVoisin 13 NbSommets 183 NbFaces 318 NbElemDist 4073
  Joint 3 PeVoisin 15 NbSommets 13 NbFaces 16 NbElemDist 152
               Total:    NbSommets 279 NbFaces 449 NbElemDist 6253
Writing part 12 into the binary file box_0012.Zones
Writing of 1959 nodes.
Writing of 9352 elements.
 Construction of part number 13
  Number of nodes    : 1983
  Number of elements : 9352
  Number of joints   : 4
  Joint 0 PeVoisin 10 NbSommets 82 NbFaces 133 NbElemDist 2011
  Joint 1 PeVoisin 11 NbSommets 22 NbFaces 29 NbElemDist 479
  Joint 2 PeVoisin 12 NbSommets 183 NbFaces 318 NbElemDist 3672
  Joint 3 PeVoisin 15 NbSommets 127 NbFaces 215 NbElemDist 2485
               Total:    NbSommets 414 NbFaces 695 NbElemDist 8647
Writing part 13 into the binary file box_0013.Zones
Writing of 1983 nodes.
Writing of 9352 elements.
 Construction of part number 14
 Adding of a new joint : 10
  Number of nodes    : 1934
  Number of elements : 9352
  Number of joints   : 6
  Joint 0 PeVoisin 2 NbSommets 116 NbFaces 200 NbElemDist 2352
  Joint 1 PeVoisin 5 NbSommets 80 NbFaces 134 NbElemDist 1629
  Joint 2 PeVoisin 8 NbSommets 57 NbFaces 87 NbElemDist 1394
  Joint 3 PeVoisin 10 NbSommets 0 NbFaces 0 NbElemDist 17
  Joint 4 PeVoisin 11 NbSommets 1 NbFaces 0 NbElemDist 58
  Joint 5 PeVoisin 15 NbSommets 157 NbFaces 273 NbElemDist 3188
               Total:    NbSommets 411 NbFaces 694 NbElemDist 8638
Writing part 14 into the binary file box_0014.Zones
Writing of 1934 nodes.
Writing of 9352 elements.
 Construction of part number 15
  Number of nodes    : 1977
  Number of elements : 9352
  Number of joints   : 6
  Joint 0 PeVoisin 8 NbSommets 47 NbFaces 64 NbElemDist 933
  Joint 1 PeVoisin 10 NbSommets 119 NbFaces 198 NbElemDist 2622
  Joint 2 PeVoisin 11 NbSommets 36 NbFaces 53 NbElemDist 1049
  Joint 3 PeVoisin 12 NbSommets 13 NbFaces 16 NbElemDist 266
  Joint 4 PeVoisin 13 NbSommets 127 NbFaces 215 NbElemDist 2433
  Joint 5 PeVoisin 14 NbSommets 157 NbFaces 273 NbElemDist 3335
               Total:    NbSommets 499 NbFaces 819 NbElemDist 10638
Writing part 15 into the binary file box_0015.Zones
Writing of 1977 nodes.
Writing of 9352 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
