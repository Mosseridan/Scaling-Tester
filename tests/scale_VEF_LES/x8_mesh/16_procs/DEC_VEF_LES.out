-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF_LES
 
Reading data file 

13 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
27 MBytes of RAM taken by the calculation.
28 MBytes of RAM taken by the calculation.
30 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 16
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[6.25e-02]   1=[6.25e-02]   2=[6.25e-02]   3=[6.25e-02]   4=[6.25e-02]
        5=[6.25e-02]   6=[6.25e-02]   7=[6.25e-02]   8=[6.25e-02]   9=[6.25e-02]
       10=[6.25e-02]  11=[6.25e-02]  12=[6.25e-02]  13=[6.25e-02]  14=[6.25e-02]
       15=[6.25e-02]
   Allowed maximum load imbalance: 1.001 

    149632     593168     593168 [20] [    11222:  149632 ]
     80274     423672     454452 [20] [    11222:  149632 ]
     43247     282772     363584 [20] [    11222:  149632 ]
     23078     182794     289092 [20] [    11222:  149632 ]
     12324     112200     234674 [20] [    11222:  149632 ]
      6492      64302     192842 [20] [    11222:  149632 ]
      3408      34622     156602 [20] [    11222:  149632 ]
      1781      18426     124200 [20] [    11222:  149632 ]
       928       9950      96316 [20] [    11222:  149632 ]
       486       5434      75202 [20] [    11222:  149632 ]
       252       2890      61380 [20] [    11222:  149632 ]
       132       1508      50640 [20] [    11222:  149632 ]
        68        728      41476 [20] [    11222:  149632 ]
        35        316      33076 [20] [    11222:  149632 ]
        19        144      25856 [20] [    11222:  149632 ]
     74816     295864     295864 [20] [     5611:   74816 ]
     40134     211314     226500 [20] [     5611:   74816 ]
     21581     140396     181208 [20] [     5611:   74816 ]
     11503      90346     143830 [20] [     5611:   74816 ]
      6145      55348     116606 [20] [     5611:   74816 ]
      3242      31628      95738 [20] [     5611:   74816 ]
      1705      16946      77322 [20] [     5611:   74816 ]
       895       9018      61252 [20] [     5611:   74816 ]
       464       4880      47118 [20] [     5611:   74816 ]
       243       2706      37382 [20] [     5611:   74816 ]
       127       1416      30280 [20] [     5611:   74816 ]
        66        696      24412 [20] [     5611:   74816 ]
        34        332      19646 [20] [     5611:   74816 ]
        17        132      15596 [20] [     5611:   74816 ]
     37408     147410     147410 [20] [     2805:   37408 ]
     20084     105254     112762 [20] [     2805:   37408 ]
     10820      69974      90218 [20] [     2805:   37408 ]
      5778      45026      71494 [20] [     2805:   37408 ]
      3081      27486      57858 [20] [     2805:   37408 ]
      1633      15534      47330 [20] [     2805:   37408 ]
       856       8284      38372 [20] [     2805:   37408 ]
       453       4306      30126 [20] [     2805:   37408 ]
       237       2292      22872 [20] [     2805:   37408 ]
       123       1262      17924 [20] [     2805:   37408 ]
        64        636      14240 [20] [     2805:   37408 ]
        33        302      11234 [20] [     2805:   37408 ]
        17        144       9040 [20] [     2805:   37408 ]
     18704      73240      73240 [20] [     1402:   18704 ]
     10044      52436      55920 [20] [     1402:   18704 ]
      5397      34554      44750 [20] [     1402:   18704 ]
      2874      22066      35314 [20] [     1402:   18704 ]
      1548      13376      28388 [20] [     1402:   18704 ]
       822       7498      23168 [20] [     1402:   18704 ]
       433       3930      18628 [20] [     1402:   18704 ]
       231       2066      14442 [20] [     1402:   18704 ]
       124       1086      10738 [20] [     1402:   18704 ]
        64        560       8412 [20] [     1402:   18704 ]
        34        286       6800 [20] [     1402:   18704 ]
        18        134       5294 [20] [     1402:   18704 ]
     18704      73354      73354 [20] [     1402:   18704 ]
     10025      52082      55996 [20] [     1402:   18704 ]
      5404      34556      44676 [20] [     1402:   18704 ]
      2905      22282      35440 [20] [     1402:   18704 ]
      1549      13568      28574 [20] [     1402:   18704 ]
       821       7674      23314 [20] [     1402:   18704 ]
       432       4074      18742 [20] [     1402:   18704 ]
       226       2090      14562 [20] [     1402:   18704 ]
       118       1098      10958 [20] [     1402:   18704 ]
        61        560       8192 [20] [     1402:   18704 ]
        31        262       6478 [20] [     1402:   18704 ]
        16        116       4864 [20] [     1402:   18704 ]
     37408     147386     147386 [20] [     2805:   37408 ]
     20063     105022     112696 [20] [     2805:   37408 ]
     10775      69638      90002 [20] [     2805:   37408 ]
      5744      44724      71372 [20] [     2805:   37408 ]
      3069      27262      57778 [20] [     2805:   37408 ]
      1631      15600      47290 [20] [     2805:   37408 ]
       866       8402      38228 [20] [     2805:   37408 ]
       459       4468      30580 [20] [     2805:   37408 ]
       239       2378      23660 [20] [     2805:   37408 ]
       126       1300      18636 [20] [     2805:   37408 ]
        66        684      14930 [20] [     2805:   37408 ]
        33        316      11812 [20] [     2805:   37408 ]
        17        144       9286 [20] [     2805:   37408 ]
     18704      73110      73110 [20] [     1402:   18704 ]
     10033      52030      55768 [20] [     1402:   18704 ]
      5422      34396      44486 [20] [     1402:   18704 ]
      2888      21982      35144 [20] [     1402:   18704 ]
      1545      13196      28330 [20] [     1402:   18704 ]
       819       7358      23068 [20] [     1402:   18704 ]
       440       3904      18440 [20] [     1402:   18704 ]
       230       1996      14474 [20] [     1402:   18704 ]
       120       1010      10806 [20] [     1402:   18704 ]
        64        532       8290 [20] [     1402:   18704 ]
        34        278       6478 [20] [     1402:   18704 ]
        17        126       5072 [20] [     1402:   18704 ]
     18704      73338      73338 [20] [     1402:   18704 ]
     10048      52188      56026 [20] [     1402:   18704 ]
      5426      34752      44690 [20] [     1402:   18704 ]
      2898      22204      35442 [20] [     1402:   18704 ]
      1545      13442      28580 [20] [     1402:   18704 ]
       824       7586      23368 [20] [     1402:   18704 ]
       434       4084      18744 [20] [     1402:   18704 ]
       227       2132      14642 [20] [     1402:   18704 ]
       119       1126      11058 [20] [     1402:   18704 ]
        63        602       8548 [20] [     1402:   18704 ]
        33        292       6676 [20] [     1402:   18704 ]
        17        128       5490 [20] [     1402:   18704 ]
     74816     295682     295682 [20] [     5611:   74816 ]
     40142     210770     226334 [20] [     5611:   74816 ]
     21627     140260     180714 [20] [     5611:   74816 ]
     11547      90600     143518 [20] [     5611:   74816 ]
      6171      55518     116324 [20] [     5611:   74816 ]
      3269      31664      95438 [20] [     5611:   74816 ]
      1717      17078      77022 [20] [     5611:   74816 ]
       901       9080      61178 [20] [     5611:   74816 ]
       472       4888      47292 [20] [     5611:   74816 ]
       249       2668      37138 [20] [     5611:   74816 ]
       130       1434      30366 [20] [     5611:   74816 ]
        68        724      25422 [20] [     5611:   74816 ]
        37        366      20532 [20] [     5611:   74816 ]
        19        172      16338 [20] [     5611:   74816 ]
     37408     147324     147324 [20] [     2805:   37408 ]
     20036     104758     112580 [20] [     2805:   37408 ]
     10803      69592      89842 [20] [     2805:   37408 ]
      5774      44996      71202 [20] [     2805:   37408 ]
      3087      27294      57672 [20] [     2805:   37408 ]
      1634      15566      47318 [20] [     2805:   37408 ]
       856       8300      38144 [20] [     2805:   37408 ]
       453       4434      30288 [20] [     2805:   37408 ]
       236       2376      23368 [20] [     2805:   37408 ]
       120       1270      18206 [20] [     2805:   37408 ]
        63        678      14814 [20] [     2805:   37408 ]
        33        328      12062 [20] [     2805:   37408 ]
        18        144       9282 [20] [     2805:   37408 ]
     18704      73198      73198 [20] [     1402:   18704 ]
     10043      51948      55876 [20] [     1402:   18704 ]
      5423      34396      44550 [20] [     1402:   18704 ]
      2909      22152      35228 [20] [     1402:   18704 ]
      1561      13490      28456 [20] [     1402:   18704 ]
       828       7554      23142 [20] [     1402:   18704 ]
       435       4014      18528 [20] [     1402:   18704 ]
       229       2044      14428 [20] [     1402:   18704 ]
       120       1034      10692 [20] [     1402:   18704 ]
        63        530       7992 [20] [     1402:   18704 ]
        34        268       6278 [20] [     1402:   18704 ]
        18        122       5054 [20] [     1402:   18704 ]
     18704      73294      73294 [20] [     1402:   18704 ]
     10043      51978      55972 [20] [     1402:   18704 ]
      5433      34558      44578 [20] [     1402:   18704 ]
      2910      22114      35202 [20] [     1402:   18704 ]
      1554      13416      28356 [20] [     1402:   18704 ]
       824       7636      23106 [20] [     1402:   18704 ]
       434       4032      18586 [20] [     1402:   18704 ]
       226       2134      14666 [20] [     1402:   18704 ]
       120       1102      11146 [20] [     1402:   18704 ]
        65        580       8582 [20] [     1402:   18704 ]
        34        278       6672 [20] [     1402:   18704 ]
        17        120       5046 [20] [     1402:   18704 ]
     37408     147000     147000 [20] [     2805:   37408 ]
     20088     104856     112360 [20] [     2805:   37408 ]
     10808      69402      89630 [20] [     2805:   37408 ]
      5769      44642      70878 [20] [     2805:   37408 ]
      3083      27120      57344 [20] [     2805:   37408 ]
      1630      15338      46866 [20] [     2805:   37408 ]
       849       8134      37642 [20] [     2805:   37408 ]
       440       4192      29748 [20] [     2805:   37408 ]
       231       2186      22720 [20] [     2805:   37408 ]
       123       1222      17628 [20] [     2805:   37408 ]
        63        620      13716 [20] [     2805:   37408 ]
        35        304      11128 [20] [     2805:   37408 ]
        18        128       9300 [20] [     2805:   37408 ]
     18704      73258      73258 [20] [     1402:   18704 ]
     10024      51988      55898 [20] [     1402:   18704 ]
      5415      34508      44546 [20] [     1402:   18704 ]
      2909      22244      35394 [20] [     1402:   18704 ]
      1567      13640      28586 [20] [     1402:   18704 ]
       834       7702      23446 [20] [     1402:   18704 ]
       437       4034      18936 [20] [     1402:   18704 ]
       229       2074      14692 [20] [     1402:   18704 ]
       120       1072      11194 [20] [     1402:   18704 ]
        64        570       8634 [20] [     1402:   18704 ]
        34        274       6902 [20] [     1402:   18704 ]
        19        138       5382 [20] [     1402:   18704 ]
     18704      73280      73280 [20] [     1402:   18704 ]
     10050      52364      55972 [20] [     1402:   18704 ]
      5408      34488      44784 [20] [     1402:   18704 ]
      2881      22076      35410 [20] [     1402:   18704 ]
      1540      13382      28554 [20] [     1402:   18704 ]
       817       7496      23360 [20] [     1402:   18704 ]
       433       3992      18698 [20] [     1402:   18704 ]
       226       2050      14622 [20] [     1402:   18704 ]
       119       1068      11090 [20] [     1402:   18704 ]
        62        566       8348 [20] [     1402:   18704 ]
        33        294       6658 [20] [     1402:   18704 ]
        18        146       5424 [20] [     1402:   18704 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.949
     Coarsening: 		   0.635
            Matching: 			   0.288
            Contract: 			   0.307
     Initial Partition: 	   0.004
     Uncoarsening: 		   0.220
          Refinement: 			   0.129
          Projection: 			   0.089
     Splitting: 		   0.082
********************************************************************

 gk_mcore statistics
           coresize:      1796616         nmops:         2048  cmop:      0
        num_callocs:         1884   num_hallocs:            0
       size_callocs:     33848768  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:      1795600   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

113 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 9.07442 s
